source/agora/node/Config.d

```diff
@@ -463,9 +463,20 @@ public struct RegistryConfig
                    "registry.{}: Authoritative zones require both 'primary' and " ~
                    "'email' fields to be set", name);
 
-            // If we're not primary, there's no configuration to validate
-            if (!zone.primary.set)
+            // Secondary registry
+            if (!zone.primary.set && zone.authoritative.set)
+            {
+                ensure(zone.primary_servers.length > 0,
+                    "registry.{} is secondary and at least one primary is required",
+                    name);
+
+                ensure(zone.redirect_primary.length > 0,
+                    "registry.{} is secondary (authoritative), thus requires an " ~
+                    "address registering endpoint to an Agora node",
+                    name);
```

Comment:
Perhaps we should make it default to `primary` ? It seems that in most configuration, that'd be the same.